extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/FormAuthRuntimeConfig.java

```diff
@@ -0,0 +1,122 @@
+package io.quarkus.vertx.http.runtime;
+
+import java.time.Duration;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * config for the form authentication mechanism
+ */
+@ConfigGroup
+public class FormAuthRuntimeConfig {
```

Comment:
I don't think adding a new config class in a maintenance branch is what we want to do.

Reply:
Fixed. Thanks again for catching this error.